medusa/history.py

```diff
@@ -25,7 +25,8 @@
 from .show.history import History
 
 
-def _logHistoryItem(action, showid, season, episode, quality, resource, provider, version=-1, proper_tags=''):
+def _logHistoryItem(action, showid, season, episode, quality, resource,
+                    provider, version=-1, proper_tags='', manual_searched=False):
```

Comment:
@medariox other functions use this: log subtitle, log download, etc...
only log snatch will send True - if true